From 605b2cbe5cebd649fad5040532c246935d5b21f7 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 11:03:59 +0800
Subject: [PATCH] Test case3-E for Change BuildOptions.IA32.EDKII.PEIM

Description: Change build option under BuildOptions.IA32.EDKII.PEIM (Add "XCODE:*_*_*_DLINK_FLAGS = -seg1addr 0x1000 -segalign 0x1000") in TestPkg/buildoptions.dsc.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/buildoptions.dsc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/TestPkg/buildoptions.dsc b/TestPkg/buildoptions.dsc
index 9ca9e101ff..7a220e52ea 100644
--- a/TestPkg/buildoptions.dsc
+++ b/TestPkg/buildoptions.dsc
@@ -37,6 +37,7 @@
 
 [BuildOptions.IA32.EDKII.PEIM]
   GCC:*_*_*_DLINK_FLAGS = -z common-page-size=0x1000
+  XCODE:*_*_*_DLINK_FLAGS = -seg1addr 1000 -segalign 0x1000
   
 [BuildOptions.common.EDKII.BASE]
   MSFT:RELEASE_*_*_CC_FLAGS            = /D MDEPKG_NDEBUG
\ No newline at end of file
-- 
2.27.0.windows.1

